content/fr/edge-computing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Edge Computing
+status: Completed
+category: Technology
+---
+
+L'Edge Computing est une [architecture distribuée](/fr/distributed-systems/) qui déplace une partie de la capacité de stockage et de calcul du centre de données principal vers la source de données.
+Les données collectées sont traitées localement (par exemple, dans une usine, dans un magasin ou depuis une ville) plutôt que d'être envoyées à un centre de données centralisé pour traitement et analyse. 
+Ces unités ou dispositifs de traitement locaux représentent la bordure du système, tandis que le centre de données en est le centre. 
+L'information calculée par ces unités péripheriques est ensuite renvoyée au centre de données principal pour un traitement ultérieur. 
+Parmi les exemples de Edge Computing, nous avons les bracelets connectés ("wrists gadgets"), ou les ordinateurs qui analysent le flux de circulation.
```

Comment:
Compliqué de trouvé un équivalent a "wrists gadgets" en français.  
J'ai mis  bracelets connectés en attendant vos suggestions. 
VO : _Examples of edge computing include wrists gadgets or computers that analyze traffic flow._

Reply:
Bracelet connecté me parait être une bonne traductions

Reply:
est-ce que je laisse le terme  "wrists gadgets" entre parenthèses dans FR par securité ?
Votre avis ?

Reply:
Ok, pour l'enlever aussi